sdk/attestation/azure-security-attestation/azure/security/attestation/_common.py

```diff
@@ -14,18 +21,93 @@
 
 import base64
 
-class Base64Url:
-    """Equivalent to base64.urlsafe_b64encode, but strips padding from the encoded and decoded strings.
+
+def base64url_encode(unencoded):
+    # type: (bytes)->str
```

Comment:
need space before/after the '->'

Reply:
Does anything care about this? Or is it a style thing?

If the spacing is critical, why doesn't black correct it (it does fix other spacing issues, so I would think this is within its charter)?